### ISSUE TITLE
Replacing BA proxmark by AL proxmark3?

### ISSUE THREAD

**Issue by D3vil0p3r** [BlackArch/blackarch#4302]
## Discussion

Currently BA [proxmark](https://github.com/BlackArch/blackarch/blob/master/packages/proxmark/PKGBUILD) package refers to a [source repo](https://github.com/Proxmark/proxmark3) with latest commit on 30 Mar 2021.

[AL repository](https://archlinux.org/packages/extra/x86_64/proxmark3/) has [proxmark3](https://github.com/RfidResearchGroup/proxmark3) that is the Iceman fork of the above one, and the last release is on 10 Sept 2024 (despite the AL package is flagged as outdated).

Should BA still keep the package?

**Comment by D3vil0p3r** [comment]
@noraj can you show me your POV? It is inside [broken](https://github.com/BlackArch/blackarch/blob/master/lists/broken) list. Should its proxmark/PKGBUILD file to be removed from BA repository? I get some free time to give a look to the remaining opened BA issue tickets. Thanks

**Comment by noraj** [comment]
I mean yeah D3vil0p3r seems right. But I'm not very familiar with the package removal process. @noptrix should know what to do.

**Comment by D3vil0p3r** [comment]
I guess that it is just needed to add the package inside [to-remove](https://github.com/BlackArch/blackarch/blob/master/lists/to-remove). I guess noptrix implemented some CI/CD workflow that actually removes all packages declared in that list.

**Comment by noraj** [comment]
I don't think that is automatic but more a reminder. Also I think the PKGBUILD are kept as archive while the tarball is removed from the repository.

**Comment by D3vil0p3r** [comment]
The related PR has been merged. I guess we can close this issue.
